mongo/src/main/java/com/redhat/lightblue/mongo/crud/MongoCRUDController.java

```diff
@@ -1108,4 +1107,29 @@ private Error analyzeException(Exception e, final String otherwise, final String
             return Error.get(otherwise, msg);
         }
     }
-}
+
+    @Override
+    public CRUDHealth checkHealth() {
+        boolean isHealthy = true;
+        Collection<MongoConfiguration> configs = dbResolver.getConfigurations();
+        List<String> details = new ArrayList<>(configs.size());
+        DBObject ping = new BasicDBObject("ping", 1);
+
+        for (MongoConfiguration config : configs) {
+            try {
+                CommandResult result = config.getDB().command(ping);
+                if (!result.get("ok").equals(1.0)) {
+                    isHealthy = false;
+                    details.add(new StringBuilder(config.toString()).append("\n").append("ping:NOK").toString());
```

Comment:
Well, I did this because I planned to add MongoConfiguration details in the health check details which has following pattern in toString implementation (an excerpt) -
append("database:").append(database).append('\n').
                append("ssl:").append(ssl).append('\n')

However, I would change it use "=>" to display more elegantly in JSON and would not display the entire config details but just the mongo server details and the database name.